doc/src/Commands_compute.rst

```diff
@@ -1,27 +1,13 @@
-.. table_from_list::
```

Comment:
FYI these repeated lists were a special request by @sjplimp.
You could prefix all but the first with `.. only:: html` since this was for the HTML version only.

Reply:
I know, but they're beyond obsolete and duplicating the exact same navigation that is to the very left in the HTML version. Unless @sjplimp insists on keeping them, I would remove them.

Reply:
![lammps-docs-discussion](https://github.com/user-attachments/assets/3e74dcf0-11df-46ad-8c84-cec3294e76c7)

Reply:
In the HTML version it is redundant. In the PDF version it is more than 10 times the same table with no real purpose. So option A (as Axel suggests) is only show it in the HTML version, Option B (my preference) is get rid of the redundancy completely.